packages/button/stories/Button.stories.tsx

```diff
@@ -172,12 +172,32 @@ export const Minimal: Story = { args: { children: 'Minimal', kind: 'minimal' } }
 
 export const Primary: Story = { args: { children: 'Primary', kind: 'primary' } };
 
+export const Close: Story = { args: { children: 'Close', kind: 'close' } };
+
 export const Destructive: Story = {
   args: { children: 'Destructive', kind: 'destructive' },
 };
 
 export const Link: Story = { args: { children: 'Link', kind: 'link' } };
 
+export const PrimaryFlair: Story = { args: { children: 'Primary flair', kind: 'primaryFlair' } };
+
+export const PrimaryFlairWithIcon: Story = {
```

Comment:
The buttons with icons are not required but its a good check to confirm our icons with with the buttons